docs/developer-docs/latest/developer-resources/database-apis-reference/rest/relations.md

```diff
@@ -0,0 +1,334 @@
+---
+title: Reordering relations
+description: Use the REST API to manage the order of relations
+canonicalUrl: https://docs.strapi.io/developer-docs/latest/developer-resources/database-apis-reference/rest/relations-reordering.html
+---
+
+<style lang="scss" scoped>
+@media (min-width: 1536px) {
+  .custom-block.api-call > .response {
+    flex: 0 0 32%;
+  }
+}
+</style>
+
+# Managing relations through the REST API <BetaBadge />
+
+Relations between content-types can be managed through the [admin panel](/user-docs/latest/content-manager/managing-relational-fields.md#managing-multiple-choices-relational-fields) or through [REST](/developer-docs/latest/developer-resources/database-apis-reference/rest-api.md) requests sent to the Content API.
+
+Relations can be added, updated, or removed through the Content API by passing parameters in the body of the request:
```

Comment:
Can we be more consistent with the naming of the API here? 
```suggestion
Relations can be connected, updated, or disconnected through the Content API by passing parameters in the body of the request:
```

Reply:
I'm definitely for more consistency, but I was wondering: is it obvious to everyone that "connecting" a relation and "adding" it is the same? I was not sure, so I decided to use the "simpler" wording first. WDYT?

Edit: Seeing your other suggestions in combination with this one makes sense, I'm just still wondering what is the easiest to understand: "add a relation" or "connect an entity"? 🤔

Reply:
For me a relation is a connection between two entities - in that sense you are *creating* a connection between two things. With this mindset it is more about "creating/ connecting" entities rather than "adding" relations (where are they being added to?).

But I don't have a strong opinion on that.

Does it make sense to have a look at https://www.prisma.io/docs/concepts/components/prisma-schema/relations? They were our source when deciding on the naming.

Reply:
Thanks for the useful link! I will read it carefully 👀

Reply:
I am wondering the same. At some point we have to explain connecting = adding a relation, that place could be in the table below.

Also connecting actually means adding & updating. So if we were going to rephrase it could be more like:
> Relations can be connected, disconnected or set through the Content API by passing parameters in the body of the request:

Reply:
One option we are missing in this list is not using any of `connect` `disconnect` `set` attributes. You can create relations by passing an array of numbers:

```js
{
  data:  {
		categories: [2, 4]
  }
}
```

If we decide to mention here, I think keeping the `added, updated, removed `would be more appropiace

Reply:
@Marc-Roig What do you think about saying we are connecting one/ more entities with each other? I find that pretty clear, but I don't have a very strong opinion either ...

Reply:
@gu-stav I do like that wording indeed. :) That sounds clear and is coherent with the naming we are using.